source/production/F0.Minesweeper.Components/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using F0.Minesweeper.Components.Abstractions;
+using F0.Minesweeper.Components.Logic.Cell;
+using F0.Minesweeper.Components.Logic.Game;
+using Microsoft.Extensions.DependencyInjection;
+using Prism.Events;
+
+namespace F0.Minesweeper.Components.Extensions
+{
+	public static class IServiceCollectionExtensions
+	{
+		public static IServiceCollection AddMinesweeperComponentsServices(this IServiceCollection serviceCollection)
+		{
+			serviceCollection.AddSingleton<IEventAggregator>(new EventAggregator());
+			serviceCollection.AddTransient(CellStatusManagerFactory.GetManager);
+			serviceCollection.AddSingleton<IGameUpdateFactory>((ServiceProvider) => new GameUpdateFactory(ServiceProvider.GetRequiredService<IEventAggregator>()));
```

Comment:
`ServiceProvider` should start with a lowercase letter here

Reply:
It's now soooooo low it's even lower than your humour.